src/main/java/ch/njol/skript/log/RedirectingLogHandler.java

```diff
@@ -30,31 +31,35 @@
 public class RedirectingLogHandler extends LogHandler {
 
 	private final CommandSender recipient;
-	
 	private final String prefix;
-	
 	private int numErrors = 0;
-	
+	private final List<String> errorMessages = new ArrayList<>();
```

Comment:
You should override `onStop` to clear this list. I do not think we want its contents to remain when the loghandler is stopped.